pages/common/json5.md

```diff
@@ -0,0 +1,7 @@
+# json5
+
+> A command-line tool for transpiling JSON5 files to JSON.
+
+- Transpile a JSON5 file to a JSON file of the same name:
+
+`json5 -c {{path/to/file}}`
```

Comment:
Hmm .. I don't see the `-c` option anywhere here - https://github.com/json5/json5#cli. What am I missing ?

Reply:
Not sure why `-c` isn't listed as that's one of the main options. :thinking: Not sure why I missed the other available commands, I'll add them too.

Reply:
Turns out my version was slightly out of date. I'll redo the page. _sigh_